what4/src/What4/Solver/CVC5.hs

```diff
@@ -224,9 +227,15 @@ setInteractiveLogicAndOptions writer = do
     SMT2.setOption writer "produce-models" "true"
     -- Tell cvc5 to make declarations global, so they are not removed by 'pop' commands
     SMT2.setOption writer "global-declarations" "true"
+    -- FIXME: in cvc5-1.0.1, unsat-cores and produce-abducts are incompatible, but
+    -- cvc5-1.0.2 will fix this, and we can uncomment this once we that is released.
+    -- Closing issue #214 will fix this.
```

Comment:
CVC5 1.0.2 was [released](https://github.com/cvc5/cvc5/releases/tag/cvc5-1.0.2) recently. We should check to see if that version fixes the issues observed here.